modules/ROOT/pages/tokenization-formats.adoc

```diff
@@ -46,6 +46,10 @@ When you xref:create-tokenization-format.adoc[create a tokenization format] and
 
 |*Lax lowercase/decimal - Preserving* 
 |Lowercase Latin characters in the clear­text remain lowercase Latin in the obfuscated­ text. Decimal digits in the clear­text remain decimal digits in the obfuscated­ text. All other characters are handled as lax characters. Preserve first and last "n" characters.
+
+|*Masking*
+|Replaces characters with a specified masking character, for example `#. The input is all printable ASCII characters. Printable ASCII characters in the optional *Ignore Characters* configurable field are ignored. The character specified in the *Masking Character* configurable option is used to mask character identities.
```

Comment:
```suggestion
|Replaces characters with a specified masking character, for example `#`. The input is all printable ASCII characters. Printable ASCII characters in the optional *Ignore Characters* configurable field are ignored. The character specified in the *Masking Character* configurable option is used to mask character identities.
```

Reply:
Shouldn't the details for "all printable ASCII characters" go into the Masking Character description?

Reply:
Fixed the typo.

Reply:
> Shouldn't the details for "all printable ASCII characters" go into the Masking Character description?

I think this sentence describes input to the Masking domain, while Ignore Characters and Masking Character are filters on that input. If I were to rewrite this topic, I would combine this information into one place.